lib/runtime/wasmer/imports_test.go

```diff
@@ -61,6 +61,35 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
+func Test_ext_offchain_timestamp_version_1(t *testing.T) {
+	t.Skip()
+	// TODO: Fix me: https://github.com/ChainSafe/gossamer/issues/1902
```

Comment:
Don't comment the code. We are already skipping the test.

Reply:
@arijitAD Commented codes were left for an immediate reference to make the fixing easier. But, noted!